cookbooks/aws-parallelcluster-slurm/recipes/test/mock_munge_key.rb

```diff
@@ -1,10 +1,13 @@
-munge_user_dir = "/home/#{node['cluster']['cluster_user']}/.munge"
-directory munge_user_dir do
-  mode '1777'
-end
+munge_dirs = %W(#{node['cluster']['shared_dir']}/.munge #{node['cluster']['shared_dir_login']}/.munge)
+
+munge_dirs.each do |munge_dir|
+  directory munge_dir do
+    mode '0700'
+  end
 
-file "#{munge_user_dir}/.munge.key" do
-  content 'munge-key'
-  owner node['cluster']['munge']['user']
-  group node['cluster']['munge']['group']
+  file "#{munge_dir}/.munge.key" do
+    content 'munge-key'
```

Comment:
If we manage to throw a binary blob in this file, we can maybe do some more interesting stuff with this mocker? Or maybe not...